examples/js/archive.js

```diff
@@ -249,6 +251,9 @@ document.addEventListener('DOMContentLoaded', async (event) => {
           let imageURL;
           let options;
       
+          // let cornerBounds = .... // collect all corners into a big array of [lat, lon]
```

Comment:
Here's where we can collect up all the corner lat/lon pairs, and make an array. Then on the next line, we fit the window to contain them all before we start placing images!